pkg/docker/docker-entrypoint.sh

```diff
@@ -1,11 +1,11 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 
 set -e
 
 curl_put()
 {
-    RET=`/usr/bin/curl -s -w '%{http_code}' -X PUT --data-binary @$1 --unix-socket /var/run/control.unit.sock http://localhost/$2`
-    RET_BODY=${RET::-3}
+    RET=$(/usr/bin/curl -s -w '%{http_code}' -X PUT --data-binary @$1 --unix-socket /var/run/control.unit.sock http://localhost/$2)
+    RET_BODY=$(printf '%s' "$RET" | cut -c 1)
```

Comment:
Can't we just have an `echo(1)` here?

Reply:
Sorry but I don't know how it exactly  works to change this

Reply:
I meant `$(echo "$RET" | cut -c 1)`.  Isn't that equivalent to what you wrote?  Why do we need printf(1)?

Reply:
If you don't know how echo works: it just prints its arguments, separated by space (but normally you just pass a signle argument, so it prints it literally).